CHANGELOG.md

```diff
@@ -74,4 +74,15 @@ task example {
     ]
   }
 }
-```
\ No newline at end of file
+```
+* Bug fix: Tasks that changed directory would fail on JES because their return code file was written to the new directory instead of an absolute path
+* Bug fix: Using `write_*` functions in a Task's command (e.g. `./my_script --file=${write_file(my_array)}`) will now work with JES
+* Changing format of the 'disks' runtime attribute slightly to allow for mounting disks at specific mountpoints
```

Comment:
it'll be great when we're using GitHub issues and can link to them in this changelog